Stephanie Vettese Handing In Assignment 2 for Medinet Gurob

Vetteses95 opened the pull request.

geyerbri committed:
Merge pull request #68 from msu-anthropology/master

Add student work from Assignment 1 into main site
Use image more relevant to a course for wiki

geyerbri committed:
Merge pull request #69 from msu-anthropology/master

update example caption

Vetteses95 committed:
Merge pull request #4 from msu-anthropology/gh-pages

Update 10/25

geyerbri committed:
Merge pull request #72 from msu-anthropology/master

move kafr-tarkhan from /AA-template to /sites

geyerbri committed:
Merge pull request #74 from msu-anthropology/master

Update gebel-el-haridi.html

Vetteses95 committed:
Merge pull request #6 from msu-anthropology/gh-pages

Update 10/27

Vetteses95 committed:
Update two added images Stephanie Vettese

Vetteses95 committed:
Update for images 10-27-16

Vetteses95 committed:
Update 10/27 with date and images

geyerbri commented:
Be advised, your work contains a few issues.

First, your image files contain uppercase letters. Please rename them and then adjust your html code to reflect the new, lowercase filenames.

Second, your second `figure` code does not have a closing tag. Please add `</figure>` to the end of Line 84 of your html file.

Vetteses95 committed:
Fixed Picture issues

Fixed the uppercase letters in the image names and added  </figure> to
my second image

geyerbri commented:
You have successfully changed your code. The issue with your file names may be a little difficult to fix, but I have an idea. Please change your "img" folder to "imgs" and then commit the name change in your GitHub application. Then, after committing, change the name back to "img" and commit it again. Doing this should trick the system into properly renaming your image files online.

Vetteses95 committed:
Renamed file from omg to imgs

Vetteses95 committed:
Changed folder back to img

Vetteses95 committed:
Took folder out

Vetteses95 committed:
Changed "img" folder to "imgs"

geyerbri commented:
Thank you.
